index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <title>PortfolioPage</title>
+  </head>
+  <body>
+    <header>
+      <div class="logo-placeholder"><a href="#">Rich</a></div>
+      <nav class="menu"><img src="./logos/list.png" alt="menu" /></nav>
```

Comment:
The menu icon used here is different from the one used in the original design. Kindly use a similar one or simply download the original one from the Figma template. This will make your portfolio look more like the original template.

- Also give some margin and padding so it's placed like in the Figma design

Difference between the original and yours below

- Your Design
![image](https://user-images.githubusercontent.com/81932478/164537107-1b69230d-348e-4080-9ea2-12bd60271e50.png)

- Figma Design
![image](https://user-images.githubusercontent.com/81932478/164537301-f5a019d6-7325-4d86-8cff-15caf8bffe40.png)